### PR TITLE
Fix buffer number (#91)

### PR DESCRIPTION
**Pull request by bimlas**
0 is the alternate buffer, not the current one.

### CODE REVIEW

**Comment by bimlas** [comment]
As I know it works. Steps of testing:

```
:set signcolumn=auto
:Gina blame :
:sign define piet text=>> texthl=Search | exe ":sign place 2 line=23 name=piet file=" . expand("%:p")
:exe ":sign place 2 line=23 name=piet file=" . expand("%:p")
Press <C-L> on navi
```
(the sign example is taken from `:help :sign`)

File: autoload/gina/command/blame.vim [hunk 1]
```diff
@@ -265,7 +265,7 @@ endfunction
 
 function! s:is_sign_visible() abort
   if !exists('&signcolumn') || &signcolumn ==# 'auto'
-    return len(split(execute('sign place buffer=0'), '\r\?\n')) > 1
+    return len(split(execute('sign place buffer='.bufnr('%')), '\r\?\n')) > 1
```
Comment (lambdalisue) [inline autoload/gina/command/blame.vim]: Please add spaces around `.` while Vim use `.` for 1) concatenating strings 2) dictionary attribute. Without spaces, it is a bit difficult to distinguish

**Review by lambdalisue — CHANGES_REQUESTED** [review]

**Comment by bimlas** [comment]
Forced push the change.

**Comment by lambdalisue** [comment]
Thanks a lot!

**Pull request merged**